api/src/pcapi/scripts/add_pricing_points/main.py

```diff
@@ -0,0 +1,115 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions):
+
+https://github.com/pass-culture/pass-culture-main/blob/ogeber/pc-37715-add-missing-pricing-points/api/src/pcapi/scripts/add_pricing_points/main.py
+
+"""
+
+import argparse
+import csv
+import logging
+import os
+import typing
+
+import psycopg2
+
+from pcapi.app import app
+from pcapi.core.history import api as history_api
+from pcapi.core.history import models as history_models
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offerers.api import link_venue_to_pricing_point
+from pcapi.core.offerers.exceptions import CannotLinkVenueToPricingPoint
+from pcapi.models import db
+from pcapi.utils.transaction_manager import atomic
+from pcapi.utils.transaction_manager import mark_transaction_as_invalid
+
+
+logger = logging.getLogger(__name__)
+
+VENUE_ID_HEADER = "venue_id"
+SIRET_VENUE_ID_HEADER = "siret_venue_id"
+
+
+def _read_csv_file(filename: str) -> typing.Iterator[dict[str, str]]:
+    namespace_dir = os.path.dirname(os.path.abspath(__file__))
+    with open(f"{namespace_dir}/{filename}.csv", "r", encoding="utf-8") as csv_file:
+        csv_rows = csv.DictReader(csv_file, delimiter=",")
+        yield from csv_rows
+
+
+def _get_venues_to_update(filename: str) -> dict[int, offerers_models.Venue]:
+    rows = list(_read_csv_file(filename))
+
+    venue_ids = {row[VENUE_ID_HEADER] for row in rows}
+    venues_dict = {
+        venue.id: venue
+        for venue in db.session.query(offerers_models.Venue).filter(offerers_models.Venue.id.in_(venue_ids)).all()
+    }
+
+    result = {}
+    for row in rows:
+        venue_id = int(row[VENUE_ID_HEADER])
+        siret_id = int(row[SIRET_VENUE_ID_HEADER])
+        if venue_id in venues_dict:
+            result[siret_id] = venues_dict[venue_id]
+        else:
+            logger.info("Venue %s not found, not processing", venue_id)
+    return result
+
+
+def _add_action_history(
+    venue: offerers_models.Venue, old_pricing_point_id: int | None, new_pricing_point_id: int | None
+) -> None:
+    history_api.add_action(
+        venue=venue,
+        author=None,
+        action_type=history_models.ActionType.INFO_MODIFIED,
+        comment="Venue linked to pricing point",
+        extra_data={
+            "modified_info": {
+                "pricingPoint": {
```

Comment:
On n'a pas de traduction pour `"pricingPoint"` dans `pcapi.routes.backoffice.filters.format_modified_info_name`, donc tel que tu le saisis ça affichera « Pricingpoint » dans l'historique du BO. Car on enregistre en fait le changement de `"pricingPointSiret"` habituellement.

S'il n'est pas pertinent d'enregistrer `"pricingPointSiret"` dans ton cas, je suggère : 
```suggestion
                "pricingPointId": {
```
et en même temps ajouter dans `pcapi.routes.backoffice.filters.format_modified_info_name` quelque chose comme : 
```
        case "pricingPointId":
            return "ID du point de valorisation"
```